lhotse/dataset/vad.py

```diff
@@ -27,13 +27,12 @@ class VadDataset(Dataset):
     def __init__(
             self,
             cuts: CutSet,
-            duration: Optional[Seconds] = 5.0,
```

Comment:
A few lines above here: instead of `Returns a dict of:`, I'd rather say: `each item in this dataset is a dict of:`... I got confused because I thought by "Returns" you meant this class itself, which is like a dict too.